src/test/mediapipeflow_test.cpp

```diff
@@ -2822,7 +2830,7 @@ TEST_F(MediapipeFlowStartTest, AsSoonAsMediaPipeGraphDefinitionReadyInferShouldP
     auto start = std::chrono::high_resolution_clock::now();
     while (!isMpReady(servableName) &&
            (std::chrono::duration_cast<std::chrono::seconds>(std::chrono::high_resolution_clock::now() - start).count() < SERVER_START_FROM_CONFIG_TIMEOUT_SECONDS)) {
-        std::this_thread::sleep_for(std::chrono::microseconds(100));
+        std::this_thread::sleep_for(std::chrono::microseconds(600));
```

Comment:
Please check if this sleep is necessary and if it is necessary to have it this long.